docs/revsets.md

```diff
@@ -78,143 +78,140 @@ only symbols.
 You can use parentheses to control evaluation order, such as `(x & y) | z` or
 `x & (y | z)`.
 
-<details>
-<summary>Examples:</summary>
+??? examples
```

Comment:
Just to make sure, this `???` is valid MkDocs markdown (or whatever the flavor is called)?

Reply:
I got it from https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks.

I wonder whether it would work with other MkDocs themes, but it works with ours. I would guess it requires some MkDocs extension that the theme ships by default.